m4_sessions/automation/session_one/eight_mike_justin_jesse_ethan_dan.md

```diff
@@ -1,3 +1,53 @@
 ## Group Eight's Session Plan
 
 M4 Student Leaders: Mike, Justin, Jesse, Ethan, Dan
+
+# Turing Career Development: Automation
+
+## Session Outcomes
+
+## Mission Alignment		 
+  		  
+*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
+  		  
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+
+## Pre-Session
+
+
+### Survey Questions
+Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+
+#### Survey to lighten the mood
+  * funny questions like:
+   1. When the robots attack, what’s the first thing you do?
+   2. If you could have one process in your life automated tomorrow what would it be?
+   3. What automated process today do you appreciate the most
+   4. Write your apology to the person whose job you stole by automating that thing.
+   
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
```

Comment:
who will send out the pre-reading? is the expectation that they read/watch all the resources here?